tests/format/typescript/edge-cases/format.test.js

```diff
@@ -0,0 +1 @@
+runFormatTest(import.meta, ["typescript"]);
```

Comment:
Can't find a suitable dir to put this test. Better idea than `edge-cases/`?

Reply:
Sounds good